File: win-prog/ProgWinCS/Hello Windows Forms/InheritTheForm/InheritTheForm.cs
//---------------------------------------------
// InheritTheForm.cs © 2001 by Charles Petzold
//---------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class InheritTheForm: Form
{
     public static void Main()
     {
          InheritTheForm form = new InheritTheForm();
          form.Text = "Inherit the Form";
          form.BackColor = Color.White;

          Application.Run(form);
     }
}

File: win-prog/ProgWinVB/Hello Windows Forms/InheritTheForm/InheritTheForm.vb
'-----------------------------------------------
' InheritTheForm.vb (c) 2002 by Charles Petzold
'-----------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Module InhertTheForm
    Sub Main()
        Dim frm As New InheritFromForm()
        frm.Text = "Inherit the Form"
        frm.BackColor = Color.White

        Application.Run(frm)
    End Sub
End Module

Class InheritFromForm
    Inherits Form
End Class
